
<launch>
<param name="use_sim_time" value="false"/>
  <node pkg="nmea_navsat_driver" type="nmea_topic_serial_reader" name="nmea_topic_serial_reader"/>
  <node pkg="nmea_navsat_driver" type="nmea_topic_driver" name="nmea_topic_driver"/>
  <node pkg="nmea_vtg_reader" type="nmea_vtg_reader" name="nmea_vtg_reader" output="screen"/>

</launch>
